Gems/Atom/RPI/Code/Source/RPI.Builders/Pass/PassBuilder.cpp

```diff
@@ -37,7 +37,7 @@ namespace AZ
         {
             AssetBuilderSDK::AssetBuilderDesc builder;
             builder.m_name = PassBuilderJobKey;
-            builder.m_version = 12; // ATOM-15472
+            builder.m_version = 13; // antonmic: making .pass files declare dependency on shaders they reference
```

Comment:
Nit: I think our guideline suggests not adding names to comment. Can you change to the JIRA id.